compiler/ballerina-lang/src/main/java/io/ballerina/projects/ResourceConfig.java

```diff
@@ -64,11 +63,8 @@ public String name() {
         return name;
     }
 
-    public Optional<byte[]> content() {
-        if (content == null) {
-            return Optional.empty();
-        }
-        return Optional.of(content);
```

Comment:
Dont we need to keep these changes? @azinneera

Reply:
The change was done for IDL client implementation. So reverting is fine.